client/state/data-layer/wpcom/me/settings/index.js

```diff
@@ -66,13 +72,31 @@ export function saveUserSettings( action ) {
 		}
 	};
 }
+
+export function saveUserSettingsFailure( { settingsOverride }, error ) {
+	if ( settingsOverride?.password ) {
+		return [
+			errorNotice( translate( 'There was a problem saving your password. Please, try again.' ) ),
```

Comment:
Let's add a notice ID to this one. Currently, if you try several times and it fails several times, you get a duplicate notice:

![](https://cldup.com/jTJuFXsYP6.png)

Reply:
addressed in d2bf11a992113ab58aa843f7e497722be7c1570e